stable/stolon/templates/hooks/create-cluster-job.yaml

```diff
@@ -17,6 +17,10 @@ spec:
       labels:
         app: {{ template "stolon.fullname" . }}
         release: {{ .Release.Name }}
+      annotations:
```

Comment:
please wrap both places into `if` statement, since these annotations are optional:
here is the example 
https://github.com/helm/charts/blob/e5c7915aa958bc38236145db19ef60d23409f292/stable/prometheus-operator/templates/prometheus/rules-1.14/kubernetes-absent.yaml#L23-L26